terraform/warehouse/vcl/main.vcl

```diff
@@ -477,6 +477,43 @@ sub vcl_deliver {
     set resp.http.X-Content-Type-Options = "nosniff";
     set resp.http.X-Permitted-Cross-Domain-Policies = "none";
 
+    # See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy
+    set resp.http.Permissions-Policy = {"
```

Comment:
curious what will happen to the line breaks and if they'll be a problem in anyway.

Reply:
Thanks for the callout! 

https://varnish-cache.org/docs/7.3/reference/vcl.html#strings tells us that:

> They may contain any character including single double quotes ", newline and other control characters except for the NUL (0x00) character.

Although the Fastly reference doesn't repeat it: https://developer.fastly.com/reference/vcl/types/string/

Playing around with their Fiddle tool appears to confirm that these line breaks probably won't work 😭 
https://fiddle.fastly.dev/fiddle/348c77d9

I'll update to a be a single long line 🤢